pkg/output/output.go

```diff
@@ -30,13 +30,23 @@ func (o *Output) AddException(message error) {
 	o.exceptions = append(o.exceptions, message)
 }
 
-// SetFailures sets failures as a bulk update
-func (o *Output) SetFailures(failures []string) {
+// SetEgressFailures sets egress endpoint failures as a bulk update
+func (o *Output) SetEgressFailures(failures []string) {
 	for _, f := range failures {
 		o.failures = append(o.failures, handledErrors.NewEgressURLError(f))
 	}
 }
 
+// GetEgressFailures gets egress endpoint failures
+func (o *Output) GetEgressFailures() (failures []error) {
+	return o.failures
+}
+
+// GetExceptions gets generic errors
+func (o *Output) GetExceptions() (errors []error) {
```

Comment:
why do we need this? I saw the usage only in _tests_
I can recommend using `Parse()` which returns all the output params individually
```go
_, exceptions, _ := cli.output.Parse()
```